shap/utils/_clustering.py

```diff
@@ -267,5 +286,3 @@ def hclust(
         return scipy.cluster.hierarchy.complete(dist)
     elif linkage == "average":
         return scipy.cluster.hierarchy.average(dist)
-    else:
-        raise ValueError("Unknown linkage: " + str(linkage))
```

Comment:
This check is irrelevant now since we already ensure the `linkage` parameter passed in is one of the 3 acceptable values above.